beacon-chain/das/cache.go

```diff
@@ -0,0 +1,117 @@
+package das
+
+import (
+	"bytes"
+
+	errors "github.com/pkg/errors"
```

Comment:
do we need to alias here?

Reply:
huh not sure why ide added the import that way, will tidy it up.